src/core/qgsdistancearea.h

```diff
@@ -19,6 +19,7 @@
 #include "qgis_core.h"
 #include <QList>
 #include <QReadWriteLock>
+#include <QUuid>
```

Comment:
Yes, that was forgotten.
That was to test if it was still the same object.

This was the result:

After setEllipsoid all values are correctly set:
<pre>
257: src/core/qgsdistancearea.cpp: 91:
 (setEllipsoid) [0ms] QgsDistanceArea::setEllipsoid -53- : 
mEllipsoid[WGS84] 
[SemiMajor[6378137.0000000] 
SemiMinor[6356752.3142452] 
InvFlattening[298.2572236] 
uuid[{7aaeedc4-fabb-4cf7-8fc5-25fb10ff277d}]]
</pre>

During measureLineProjected all 4 values are again set to the default values
(at that point I used 0, changed later to -1 as in QgsEllipsoidUtils)
<pre>
257: src/core/qgsdistancearea.cpp: 390: 
(measureLineProjected) [0ms] 
Converted distance of 0.0000090 meters to 54.1933740 distance Geographic degrees, using 
azimuth[1.5708] from point[POINT(13.37770458660235029 52.51627178856762157)] 
to point[POINT(13.37770483284973544 52.51628076834474257)] 
sourceCrs[WGS 84,NONE] 
isGeographic[1] 
[SemiMajor[0.0000000] 
SemiMinor[0.0000000] 
InvFlattening[0.0000000] 
uuid[{7aaeedc4-fabb-4cf7-8fc5-25fb10ff277d}]]
</pre>